src/validators/divorce.test.js

```diff
@@ -136,7 +136,6 @@ describe('Divorce validation', function() {
           BirthPlace: {
             country: { value: 'United States' },
             city: 'Arlington',
-            county: 'Arlington',
             state: 'VA',
             layout: Location.BIRTHPLACE
```

Comment:
This layout will need to be updated as well to reflect `BIRTHPLACE_WITHOUT_COUNTY`